Tests/FluentAssertions.Equivalency.Specs/InlineAssertionsSpecs.cs

```diff
@@ -75,24 +80,11 @@ public void Succeeds_for_a_matching_condition()
     [Fact]
     public void A_condition_expression_is_required()
     {
-        // Arrange
-        var actual = new
-        {
-            Name = "John",
-            Age = 30
-        };
-
-        var expectation = new
-        {
-            Name = "John",
-            Age = Value.ThatMatches<int>(null)
-        };
-
-        // Act
-        var act = () => actual.Should().BeEquivalentTo(expectation);
+        // Arrange / Act
+        var act = () => Value.ThatMatches<int>(null);
```

Comment:
Although this is technically correct, the point is that `Value.ThatMatches` is supposed to be used in combination with `BeEquivalentTo`. The fact that you moved the `null` check is an implementation detail and should not affect the "surface area" of the test.  Same for the other test.